action.yml

```
@@ -76,7 +76,8 @@ inputs:
   file_triggers_enabled:
     description: Whether to filter runs based on the changed files in a VCS push
     required: false
-
+  remote_state_blocks:
```

```suggestion
  remote_states:
```

Blocks is a TF implementation detail